src/promptflow-devkit/tests/sdk_cli_test/e2etests/test_flow_save.py

```diff
@@ -418,7 +420,7 @@ def test_pf_save_callable_class(self):
 
     def test_pf_infer_signature_include_primitive_output(self):
         pf = PFClient()
-        flow_meta, _, _ = pf.flows._infer_signature(entry=global_hello, include_primitive_output=True)
+        flow_meta = pf.flows.infer_signature(entry=global_hello)
```

Comment:
the public API `pf.flows.infer_signature` is for user and shouldn't include primitive output (we don't want customer be aware of this)

Reply:
Maybe we can make `infer_signature` function & API accept `kwargs` and make `include_primitive_output` a private argument if you want to make them get the same return value.

Reply:
Add kwargs to api and function

Reply:
default behavior should still be without primitive output signature